yarn.lock

```diff
@@ -11336,6 +11343,11 @@ three-stdlib@^2.10.2:
     potpack "^1.0.1"
     zstddec "^0.0.2"
 
+three@^0.138.2:
```

Comment:
Should be replaced with peer dependency: https://github.com/sketchpunklabs/manipulator3d/issues/3